sqlobject/src/test/java/org/jdbi/v3/sqlobject/TestGetGeneratedKeysPostgres.java

```diff
@@ -83,4 +91,103 @@ public void testBatch() throws Exception {
             assertThat(dao.findNameById(ids[1])).isEqualTo("Macklin");
         });
     }
+
+    @Test
+    public void testUseRowMapperUpdate() throws Exception {
+        dbRule.getJdbi().useHandle(handle -> {
+            handle.execute("drop table something");
```

Comment:
The tests all have slightly different setup. Instead of duplicating the SQL I moved the table create to methods in each DAO. I actually like it better this way--you can tell how each setup fits into each DAO's methods.